excel/Concepts/Excel-Performance/excel-performance-and-limit-improvements.md

```diff
@@ -11,6 +11,20 @@ author: FastExcel
 
 Excel 2016 introduces new features that you can use to improve performance when you're working with large or complex Excel workbooks
 
+## VLOOKUP, HLOOKUP, MATCH improvements
+In Excel 16.0.9126.2xxx onwards VLOOKUP, HLOOKUP and MATCH for exact match on unsorted data is much faster than ever before when looking up multiple columns (or rows with HLOOKUP) from the same table range.
```

Comment:
Please change "In Excel 16.0.9126.2xxx" to "In Excel version 1809"

Reply:
Correction "In O365 version 1809 onwards, Excel's"

Reply:
@PrashSMSFT Let me know when you're done with all your updates, and I'll update the page then. Thanks.